x-pack/plugins/code/public/style/_layout.scss

```diff
@@ -0,0 +1,179 @@
+.euiFlexItem.codeContainer__adminWrapper {
+  padding: $euiSize $euiSizeXL;
+  flex-grow: 1;
+}
+
+.codeContainer__root {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.codeContainer__root--inner {
```

Comment:
there should be some naming conventions in for SCSS. This could be a good topic to share in the GAH team internal sync.

Reply:
https://elastic.github.io/eui/#/guidelines/sass

We've got documentation for naming conventions. Basically, BEM with verb states. Looking through this file it looks like it's following the correct EUI patterns, thought I'm not sure if "inner" is a modifier of root in this specific case, looks more like a related element to container/root. So probably `.codeContainer__rootInner`.